data-prepper-plugins/geoip-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/LocationData.java

```diff
@@ -0,0 +1,43 @@
+package com.amazon.dataprepper.plugins.prepper.geoip.provider;
+
+
+// This is a first pass of LocationData, simply to make sure lookup correctly passes through data.
+//TODO Find the best way to attach the location data to a document, and modify this class to allow that.
+
+import java.util.Objects;
+
+public class LocationData {
+    private final String countryName;
+    private final String subdivisionName;
+    private final String cityName;
+
+    public LocationData(String country, String subdivision, String city) {
```

Comment:
super nit - no reason to have the arg names different than the instance variable names, e.g. `String countryName`